File: test/java/test_SoilEvaporation.java
//Test generation'

import java.math.BigDecimal;
public static class Test
{
    //first);
    //test1);

    public static void test1()
    {
        double diffusionLimitedEvaporation = 6605.505;
        double energyLimitedEvaporation = 448.240;

        SoilEvaporation res0 = Estimation_SoilEvaporation.CalculateSoilEvaporation(diffusionLimitedEvaporation,energyLimitedEvaporation);
        System.out.println(" soilEvaporation: "+res0.soilEvaporation);

        System.out.println(((new BigDecimal(res0. soilEvaporation)).setScale(3, BigDecimal.ROUND_HALF_DOWN)).equals((new BigDecimal(448.240)).setScale(3, BigDecimal.ROUND_HALF_DOWN)));
    }
}

File: test/cs/test_SoilEvaporation.cs
'Test generation'

public static class Test
{
    //first);
    //test1

    public static void test1()
    {
        SoilEvaporation res0 = Estimation_SoilEvaporation.CalculateSoilEvaporation(diffusionLimitedEvaporation:6605.505,energyLimitedEvaporation:448.240);

        Console.WriteLine("{0}","soilEvaporation: "+res0.soilEvaporation);

        Console.WriteLine(" soilEvaporation Comparison: ("+Math.Round(448.240,3)+";"+Math.Round(res0. soilEvaporation,3)+") "+Equals(Math.Round(448.240,3),Math.Round(res0. soilEvaporation,3)));
    }
}
